examples/3d/fxaa.rs

```diff
@@ -0,0 +1,172 @@
+//! This examples compares MSAA (Multi-Sample Anti-Aliasing) and FXAA (Fast Approximate Anti-Aliasing).
+
+use std::f32::consts::PI;
+
+use bevy::{
+    core_pipeline::fxaa::{Fxaa, Quality},
+    prelude::*,
+    render::{
+        render_resource::{Extent3d, SamplerDescriptor, TextureDimension, TextureFormat},
+        texture::ImageSampler,
+    },
+};
+
+fn main() {
+    App::new()
+        // Disable MSAA be default
+        .insert_resource(Msaa { samples: 1 })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(toggle_fxaa)
+        .run();
+}
+
+/// set up a simple 3D scene
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+    mut images: ResMut<Assets<Image>>,
+    asset_server: Res<AssetServer>,
+) {
+    // plane
+    commands.spawn(PbrBundle {
+        mesh: meshes.add(Mesh::from(shape::Plane { size: 5.0 })),
+        material: materials.add(Color::rgb(0.3, 0.5, 0.3).into()),
+        ..default()
+    });
+
+    // cubes
+    for i in 0..5 {
+        commands.spawn(PbrBundle {
+            mesh: meshes.add(Mesh::from(shape::Cube { size: 0.25 })),
+            material: materials.add(StandardMaterial {
+                base_color_texture: Some(images.add(uv_debug_texture())),
+                ..default()
+            }),
+            transform: Transform::from_xyz(i as f32 * 0.25 - 1.0, 0.125, -i as f32 * 0.5),
+            ..default()
+        });
+    }
+
+    // Flight Helmet
+    commands.spawn(SceneBundle {
+        scene: asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0"),
+        ..default()
+    });
+
+    // light
+    const HALF_SIZE: f32 = 2.0;
+    commands.spawn(DirectionalLightBundle {
+        directional_light: DirectionalLight {
+            shadow_projection: OrthographicProjection {
+                left: -HALF_SIZE,
+                right: HALF_SIZE,
+                bottom: -HALF_SIZE,
+                top: HALF_SIZE,
+                near: -10.0 * HALF_SIZE,
+                far: 10.0 * HALF_SIZE,
+                ..default()
+            },
+            shadows_enabled: true,
+            ..default()
+        },
+        transform: Transform::from_rotation(Quat::from_euler(
+            EulerRot::ZYX,
+            0.0,
+            PI * -0.15,
+            PI * -0.15,
+        )),
+        ..default()
+    });
+
+    // camera
+    commands
+        .spawn(Camera3dBundle {
+            camera: Camera {
+                hdr: false, // Works with and without hdr
+                ..default()
+            },
+            transform: Transform::from_xyz(0.7, 0.7, 1.0)
+                .looking_at(Vec3::new(0.0, 0.3, 0.0), Vec3::Y),
+            ..default()
+        })
+        .insert(Fxaa::default());
+
+    println!("Toggle with:\n1 - NO AA\n2 - MSAA 4\n3 - FXAA (default)");
```

Comment:
Tiny suggestion, moving this at the top of the `setup()` and using `println` for each line instead of '\n' would mean you could quickly see it even without a console (for example in the wasm examples page).

Reply:
Good suggestion. Agreed.